web/css/source/_theme.less

```diff
@@ -195,6 +226,7 @@ p,li,dt,dd,td,th,details,button,form,a,div,details {
     .content-container{
       padding-left: 15%;
       display: block;
+      celear:both;
```

Comment:
What IDE are you using?  Your editor should have highlighted this for you.  If you are not seeing this highlighted then I would recommend any of the jetbrains IDEs, I use both webstorm and phpstorm.